### PR TITLE
Changes needed to run the regional workflow on WCOSS

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 
This pull request contains changes that are needed for running the latest version of the regional workflow on WCOSS.  It involves modifications to the make_grid, make_orog, and make_sfc_climo ex-scripts, and setup.sh.  There are some additional modifications required to the xml template and modulefiles, but we can discuss how to best handle those changes at a later date.  This PR needs to be tested concurrently with PR #31 of ufs-srweather-app.

## TESTS CONDUCTED: 
Successfully completed an end-to-end test on the WCOSS-Dell with these changes.

## CONTRIBUTORS (optional): 
Annette Gibbs was a huge help with running these tests on WCOSS.

### CODE REVIEW

File: scripts/exregional_make_sfc_climo.sh [hunk 1]
```diff
@@ -128,14 +128,22 @@ EOF
 #
 case $MACHINE in
 
-"WCOSS_C")
+"WCOSS_CRAY")
 # This could be wrong.  Just a guess since I don't have access to this machine.
```
Comment (JeffBeck-NOAA) [inline scripts/exregional_make_sfc_climo.sh]: @BenjaminBlake-NOAA, feel free to take out this comment now that things are correctly defined for the WCOSS_CRAY.  Thanks!

Comment (BenjaminBlake-NOAA) [reply scripts/exregional_make_sfc_climo.sh]: Good call, I missed that comment.  I just removed it!

Comment (JeffBeck-NOAA) [reply scripts/exregional_make_sfc_climo.sh]: Thanks!  Approving PR now.

**Review by JeffBeck-NOAA — APPROVED** [review]

**Comment by JeffBeck-NOAA** [comment]
Hey @BenjaminBlake-NOAA, feel free to merge this at your leisure.  We're having PR authors do the final 'squash and merge' from now on (as long as they have admin or write access).

**Comment by BenjaminBlake-NOAA** [comment]
@JeffBeck-NOAA  Sounds good I'll do that now

**Pull request merged**